examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>Now for both our qubit registers, we apply a Hadamard gate.</p><p> </p><p>For example, if you have a coin, you only have two sides: heads or tails.</p><p> </p><p> Now imagine that you apply some force to this coin and balance it so that it's neither head or tails, but in a superposition of both possible sides.</p><p> </p><p>The qubits in the first and second registers are now in a superposition of 0 and 1. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>